packages/dd-trace/src/plugin_manager.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
```

Comment:
It doesn't look like this file is tested in isolation. Is the idea that testing all plugins will indirectly test all functionality here? If so, let's at least make sure coverage is good. This is also true for a few other files in this PR.